### PR TITLE
:sparkles: Add field expire soon in Patient

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: src/nurse/urls.py [hunk 1]
```diff
@@ -27,6 +28,11 @@
         name="prescription-upload",
     ),
     path("notify/", AdminNotificationView.as_view(), name="notify"),
+    path(
+        "prescription/expire-soon/",
+        PrescriptionExpireSoonView.as_view(),
+        name="expired-soon",
+    ),
```
Comment (AndreMiras) [inline src/nurse/urls.py:35]: :eyes:

File: src/nurse/views.py [hunk 2]
```diff
@@ -54,6 +54,13 @@ class PrescriptionFileView(generics.UpdateAPIView):
     serializer_class = PrescriptionFileSerializer
 
 
+class PrescriptionExpireSoonView(APIView):
+    def get(self, request, *args, **kwargs):
+        prescriptions = Prescription.objects.expiring_soon()
+        serializer = PrescriptionSerializer(prescriptions, many=True)
+        return Response(serializer.data)
```
Comment (AndreMiras) [inline src/nurse/views.py:61]: :eyes:

File: src/nurse/urls.py (hunk 1 above)
Comment (issa-diallo) [reply src/nurse/urls.py]: it's cleaned

File: src/nurse/views.py (hunk 2 above)
Comment (issa-diallo) [reply src/nurse/views.py]: it's cleaned

**Review by AndreMiras — APPROVED** [review]
LGTM

**Pull request merged**